core/domain/collection_services.py

```diff
@@ -566,15 +566,15 @@ def get_collection_summaries_where_user_has_role(user_id):
     ]
 
 
-# TODO(bhenning): Update this function to support also matching the query to
-# explorations contained within this collection. Introduce tests to verify this
-# behavior.
-def get_collection_ids_matching_query(query_string, cursor=None):
+def get_collection_ids_matching_query(
+        query_string, categories, language_codes, cursor=None):
     """Returns a list with all collection ids matching the given search query
     string, as well as a search cursor for future fetches.
 
     Args:
         query_string: str. The search query string.
+        categories: list(str). The list of categories to query for.
+        language_codes: list(str). The list of language codes to query for.
```

Comment:
Do we need to update this docstring to include that this query will be made using OR statement?

Reply:
Done.